proposals/evaluation/index.md

```diff
@@ -0,0 +1,12 @@
+# Evaluation of proposals for Authorization
+
+| WAC | ACP |     |
```

Comment:
```suggestion
| WAC1.0 | ACP |     |
```

Reply:
I have added a commit that adds WAC+ to the table.

Reply:
If we have WAC+ do we still need WAC1.0 ? I think it doesn't hurt to have both, better safe than sorry 😸

Reply:
I guess we should ask @csarven . 
WAC and WAC+ works together and is quite minimal. 
With WAC1.0 we would need to decide whether @csarven if 1.0 makes sense.
We would then have to decide between WAC1.0+ or WAC2.0, WAC3.0 etc... 
I think there may be a number of different versions. My guess is we will be able to work out a  sequence of such stages, once we organize solutions by technical prerequisites for the solutions. 

a Definition would be:  WAC+ is any extension to WAC that is semantically compatible with the current WAC spec.

Reply:
I'm fine with WAC and WAC+ in the end we just need to distinguish the two.

Reply:
For the purpose of this document and the time being, "WAC" and "WAC+" works. It need not imply anything further (whereas versioning would).